packs/feat-effects/effect-red-gold-mortality.json

```diff
@@ -0,0 +1,43 @@
+{
+    "_id": "TpcfUI2y6Shg63V0",
+    "img": "icons/magic/death/skeleton-glow-yellow-black.webp",
+    "name": "Effect: Red-Gold Mortality",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Red-Gold Mortality]</p>\n<p>When you would regain Hit Points, you must attempt a Will save against the exemplar's class DC to determine the effects.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder War of Immortals"
+        },
+        "rules": [
+            {
+                "key": "Note",
```

Comment:
This is somewhat awkward, as the Note will only appear once the healing has been applied. Don't think there's an alternate way though.

Reply:
Yeah, it's the only way if we're gonna do it at all. Healing is easy to undo, at least.

Reply:
Eh, thinking about it, this is unacceptably clunky. We could do a note on just the damage roll with the ikon, but that seems a bit useless as there would be nothing actionable on that note. I'll probably just close this later if no better alternative comes up.

Reply:
Yeah. I like the idea, but might want to hold off until better automation options exist?